daemon/containerd/image_snapshot_windows.go

```diff
@@ -7,6 +7,14 @@ import (
 	"github.com/containerd/containerd/snapshots"
 )
 
+var (
+	remapSuffix = "-remap"
+)
```

Comment:
nit; looks like this changed to a var in the windows variant; could you make this a `const` here as well? (and while doing so make it a single line, to keep `gofumpt` happy 😅)

```suggestion
const remapSuffix = "-remap"
```